### ISSUE TITLE
Move to docker compose

### ISSUE THREAD

**Issue by matthdan** [qnib/docker-consul#1]
Don't forget to move to Docker Compte! :)
On the fig website : "Fig has been replaced by Docker Compose, and is now deprecated. The new documentation is on the Docker website."

=> mv fig.yml docker-compose.yml

Cheers!

**Comment by ChristianKniep** [comment]
yeah, alright... Thing is that the installation of compose broke my fig. But I will update. Promise! :)
